tmail-backend/combined-identity/src/main/java/com/linagora/tmail/combined/identity/CombinedUsersRepository.java

```diff
@@ -1,87 +1,27 @@
 package com.linagora.tmail.combined.identity;
 
-import java.util.Iterator;
-
 import javax.inject.Inject;
 
-import org.apache.james.core.MailAddress;
 import org.apache.james.core.Username;
-import org.apache.james.user.api.UsersRepository;
+import org.apache.james.domainlist.api.DomainList;
 import org.apache.james.user.api.UsersRepositoryException;
-import org.apache.james.user.api.model.User;
-import org.apache.james.user.cassandra.CassandraUsersDAO;
-import org.apache.james.user.ldap.ReadOnlyUsersLDAPRepository;
 import org.apache.james.user.lib.UsersRepositoryImpl;
 
-public class CombinedUsersRepository implements UsersRepository {
-
-    private final ReadOnlyUsersLDAPRepository readOnlyUsersLDAPRepository;
-    private final UsersRepositoryImpl<CassandraUsersDAO> cassandraUsersRepository;
+public class CombinedUsersRepository extends UsersRepositoryImpl<CombinedUserDAO> {
```

Comment:
I do not think we need this class at all, right?

Reply:
No, We need, because we need to override `test(Username name, String password)` method.